fluent-plugin-kubernetes_metadata_filter.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.has_rdoc      = false
 
-  gem.required_ruby_version = '>= 2.0.0'
+  gem.required_ruby_version = '>= 2.2.0'
```

Comment:
Is this intended change?
This means this plugin doesn't work with current td-agent, google-fluentd or ruby 2.1 based system.